index.js

```diff
@@ -56,10 +56,24 @@ module.exports = function(plugins, options) {
             }
 
             if (node.attrs && node.attrs.style) {
-                promise = css.process(node.attrs.style, options)
-                    .then(function(result) {
-                        node.attrs.style = result.css;
+                promise = new Promise(function(resolve, reject) {
+
+                    // First create a postcss promise
+                    var thisPromise = css.process(node.attrs.style, options)
+                        .then(function(result) {
+                            node.attrs.style = result.css;
+                        });
+                        
+                    thisPromise.then(function(res) {
+                        resolve(res);
+                    }, function(error) {
+                       if (error.name === 'CssSyntaxError' && error.reason === 'Unknown word') {
```

Comment:
I am handling it - im allowing any other PostCSS errors to fail, and only letting that one error through.